net-misc/asterisk-base/files/initd-3

```diff
@@ -0,0 +1,313 @@
+#!/sbin/openrc-run
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+extra_started_commands="forcestop reload"
+
+depend() {
+	need net
```

Comment:
"need net" is usually wrong here: https://github.com/OpenRC/openrc/blob/master/service-script-guide.md#be-wary-of-need-net-dependencies

It's not about needing network connectivity in the abstract, but rather about what practically will happen if/when the user takes down certain interfaces.

Reply:
https://github.com/jkroonza/asterisk-base/commit/b5cffcceb69686f63e5e0f7d8aa954861044cd43